src/Spinner/index.jsx

```diff
@@ -43,12 +49,15 @@ export default class Spinner extends React.Component
     {
         const {
             cssMap = createCssMap( this.context.Spinner, this.props ),
+            size
         } = this.props;
 
         return (
-            <div
+            <Icon
                 { ...attachEvents( this.props ) }
```

Comment:
This line should be removed because `attachEvents` should only be used on DOM elements (not components)

Reply:
Done.